### PR TITLE
py-awesome-slugify: new package at 1.6.5

### PR DESCRIPTION
**Pull request by codeandkey**
depends on `py-unidecode` in #15906

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-awesome-slugify/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class PyAwesomeSlugify(PythonPackage):
+    """Python flexible slugify function"""
+
+    homepage = "https://github.com/dimka665/awesome-slugify"
+    url      = "https://pypi.io/packages/source/a/awesome-slugify/awesome-slugify-1.6.5.tar.gz"
+
+    version('1.6.5', sha256='bbdec3fa2187917473a2efad092b57f7125a55f841a7cf6a1773178d32ccfd71')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-awesome-slugify/package.py:16]: Missing:
```python
depends_on('py-setuptools', type='build')
```

File: var/spack/repos/builtin/packages/py-awesome-slugify/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on('py-regex', type=('build', 'run'))
+    depends_on('py-unidecode@0.04.14:0.0499', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-awesome-slugify/package.py:18]:
```suggestion
    depends_on('py-unidecode@0.04.14:0.04.99', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-awesome-slugify/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on('py-nose', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-awesome-slugify/package.py:19]:
```suggestion
    depends_on('py-nose', type='test')
```

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-awesome-slugify/package.py]: updated

File: var/spack/repos/builtin/packages/py-awesome-slugify/package.py (hunk 2 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-awesome-slugify/package.py]: Oops -- nice catch!

File: var/spack/repos/builtin/packages/py-awesome-slugify/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-awesome-slugify/package.py:16]: added

**Comment by scheibelp** [comment]
Closing/reopening to restart tests (since dependency has since been merged)

**Pull request merged**